api/src/wfl/protocols.clj

```diff
@@ -0,0 +1,23 @@
+(ns wfl.protocols
```

Comment:
Ha! we did the same thing - perhaps move to our `jdbc` module to avoid an extra import?
https://github.com/broadinstitute/wfl/pull/386/files

Reply:
done!